.eslintrc

```diff
@@ -1,44 +1,46 @@
 {
-    "extends": ["prettier"],
```

Comment:
Is there a reason for using eslint over tslint?

Reply:
As `eslint` is now officially adopted by the TypeScript team and compatibility has improved dramatically. https://eslint.org/blog/2019/01/future-typescript-eslint

Reply:
Nice! Did not know that. Will have to switch back to eslint.